src/test/ui/process/signal-exit-status.rs

```diff
@@ -2,6 +2,7 @@
 // ignore-emscripten no processes
 // ignore-sgx no processes
 // ignore-windows
+// ignore-fuchsia code returned as ZX_TASK_RETCODE_EXCEPTION_KILL
```

Comment:
Let's put this one under the same FIXME as `out-of-stack.rs`, I think it's related enough (signal translation issue).